stock-plugins/aiverify.stock.robustness-toolbox/algorithms/robustness_toolbox/robustness_toolbox/plugin_init.py

```diff
@@ -0,0 +1,114 @@
+import argparse
+import json
+
+from robustness_toolbox.algo_init import AlgoInit
+from test_engine_core.plugins.enums.model_type import ModelType
+
+parser = argparse.ArgumentParser(
+    description="Run the plugin with specified parameters."
+)
+
+
+def run():
+    parse_input_args()
+    invoke_robustness_toolbox_plugin()
+
+
+def parse_input_args():
+    global parser
+
+    parser.add_argument("--data_path", required=True, help="Path to the data file.")
+    parser.add_argument("--model_path", required=True, help="Path to the model file.")
+    parser.add_argument(
+        "--ground_truth_path", required=True, help="Path to the ground truth data file."
+    )
+    parser.add_argument(
+        "--ground_truth",
+        required=True,
+        help="The ground truth column name in the data.",
+    )
+    parser.add_argument(
+        "--run_pipeline",
+        action=argparse.BooleanOptionalAction,
+        help="Whether to run the test as a pipeline (default: False).",
+    )
+    parser.add_argument(
+        "--model_type",
+        required=True,
+        choices=["CLASSIFICATION", "REGRESSION"],
+        help="The type of model (CLASSIFICATION or REGRESSION).",
+    )
+    parser.add_argument(
+        "--core_modules_path",
+        default="",
+        help="Path to the core modules (default: empty).",
+    )
+    parser.add_argument(
+        "--plugin_argument_values",
+        type=parse_plugin_argument_values,
```

Comment:
Can we add the actual options as CLI arguments instead? It's very hard to know what to pass in to the plugin this way with no description and help messages. I took a quick look at the various plugins and it does not seem that there are too many options so I think we can afford to be more descriptive.